templates/common/_base/files/wait-for-ipsec-connect.yaml

```diff
@@ -0,0 +1,50 @@
+mode: 0755
+path: "/usr/local/bin/ipsec-connect-wait.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -x
+
+    if [ ! -e "/etc/ipsec.d/openshift.conf" ]; then
+      exit 0
+    fi
+
+    # Modify existing IPsec connection entries with "auto=start"
+    # option and restart ipsec systemd service. This helps to
+    # establish IKE SAs for the existing IPsec connections with
+    # peer nodes. This option will be deleted from connections
+    # once ovs-monitor-ipsec process spinned up on the node by
+    # ovn-ipsec-host pod, but still it won't reestablish IKE SAs
+    # again with peer nodes, so it shouldn't be a problem.
+    if ! grep -q "auto=start" /etc/ipsec.d/openshift.conf; then
+      sed -i '/^.*conn ovn.*$/a\    auto=start' /etc/ipsec.d/openshift.conf
+    fi
+    # TODO: remove this line once review and testing are complete.
+    cat /etc/ipsec.d/openshift.conf
+    chroot /proc/1/root ipsec restart
+
+
+    # Wait for upto 3mins to get IPsec SAs to establish with peer nodes.
+    timeout=180
```

Comment:
as we talked about we need to figure out what is a suitable value here

Reply:
@trozet @igsilya we have just tested with 32 nodes (3 control plane and 29 worker nodes cluster), noticed majority of nodes takes only about 2s or 4s for IPsec SAs to get established with peer nodes and only one node took about 14s. So shall we go ahead with 60s as timeout value for now ?

cc @huiran0826

Reply:
updated the timeout with 60s, hope that's reasonable enough.